systers_portal/static/css/style.css

```diff
@@ -134,6 +134,23 @@ body > .container {
   }
 }
 
+
+/* Common: about us
+-------------------------------------------------- */
+.contact-text {
+ font-family: Georgia, "Times New Roman", Times, serif;
+ font-size: 1.2em;
+ line-height: 1.8em;
+ margin: 20px 0;
+}
+
+.contact-icon{
+  margin: 0 5px;
+}
+
+.contact-social a{
```

Comment:
And a space here before `{`.
Also add a hover effect to the links, maybe a darker color.